spec/lib/cyclonedx/report_node_modules_cyclonedx_spec.rb

```diff
@@ -281,5 +281,89 @@
         ]
       )
     end
+
+    it 'should report all the deps in the bower.json if present' do
+      repo = Salus::Repo.new('spec/fixtures/report_node_modules/bower_json')
+      scanner = Salus::Scanners::ReportNodeModules.new(repository: repo, config: {})
+      scanner.run
+
+      node_cyclonedx = Cyclonedx::ReportNodeModules.new(scanner.report)
+      expect(node_cyclonedx.build_components_object).to match_array(
```

Comment:
Please check if the cyclonedx result can be validated against the schema.  Just need to make sure `validate_cyclonedx` doesn't throw an exception.
See https://github.com/coinbase/salus/blob/f0cb8ebc8743f020da5719d6da1c9779e4d11dc2/lib/cyclonedx/report.rb#L61

Reply:
Same for the other test case.